rdrf/rdrf/models/definition/models.py

```diff
@@ -1353,30 +1353,19 @@ class EmailTemplate(models.Model):
     description = models.TextField()
     subject = models.CharField(max_length=50)
     body = models.TextField()
+    default_for_notification = models.CharField(max_length=100, choices=EMAIL_NOTIFICATIONS, null=True, default=None)
```

Comment:
This should also have 'blank=True' and you don't need `default=None` which is the default.

Reply:
When I tried using `blank` rather than `null`, there was an not-null constraint exception when creating notifications without a `default_for_notification` value.